Gemfile

```diff
@@ -8,8 +8,9 @@ ruby '2.5.0'
 # Gems to instalL: faker, shoulda matchers
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.1'
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3'
+# Use sqlite3 and postgres as the database for Active Record
+gem 'sqlite3' # development and testing
+gem 'pg'      # production
```

Comment:
Should this gem not go into the 'production' group? It is not needed for testing or development environments, right?